common/src/main/java/dev/latvian/kubejs/item/OldItemTooltipEventJS.java

```diff
@@ -10,6 +11,10 @@
 /**
  * @author LatvianModder
  */
+@Deprecated
+@KubeJSEvent(
+		client = { "client.item_tooltip" }
```

Comment:
I wouldn't include deprecated events

Reply:
I removed the reference to the ID, however, that event is still being fired under that ID.
Unlike the rest of the deprecated events.